core/storage/user/gae_models.py

```diff
@@ -47,6 +47,8 @@ class UserSettingsModel(base_models.BaseModel):
     last_agreed_to_terms = ndb.DateTimeProperty(default=None)
     # When the user last started the state editor tutorial. May be None.
     last_started_state_editor_tutorial = ndb.DateTimeProperty(default=None)  # pylint: disable=invalid-name
```

Comment:
Just wondering, can we remove the invalid-name pragma here?

Reply:
Yeah I can do it but left it out thinking another more relevant PR would be better suited for it.

Reply:
Did it anyways, so should be done.